README.md

```diff
@@ -395,10 +402,10 @@ let lastName = name.lastName
 
 * **3.1.5** Be wary of retain cycles when creating delegates/protocols for your classes; typically, these properties should be declared `weak`.
 
-* **3.1.6** Be careful when calling `self` directly from a closure as this can cause a retain cycle - use a [capture list](https://developer.apple.com/library/ios/documentation/swift/conceptual/Swift_Programming_Language/Closures.html#//apple_ref/doc/uid/TP40014097-CH11-XID_163) when this might be the case:
+* **3.1.6** Be careful when calling `self` directly from an escaping closure as this can cause a retain cycle - use a [capture list](https://developer.apple.com/library/ios/documentation/swift/conceptual/Swift_Programming_Language/Closures.html#//apple_ref/doc/uid/TP40014097-CH11-XID_163) when this might be the case:
```

Comment:
Should we recommend not using [weak self] when it wouldn't be preventing a memory leak?

I think that instead of adding [weak self] everywhere just to be safe, it would be better to have people stop, pause, and think about the object graph and whether there would actually be a leak or not. By getting into the habit of thinking about cycles in object graphs, people would be more likely to notice leaks in other situations.